### ISSUE TITLE
hyperopt.exceptions.AllTrialsFailed

### ISSUE THREAD

**Issue by pengcao** [hyperopt/hyperopt#666]
hyperopt.exceptions.AllTrialsFailed

**Comment by eliwchen** [comment]
I encountered the same problem, have you solved it?

**Comment by RayTsui** [comment]
I met the same issue when I use KerasRegression to wrap simple keras 3-layer nn written by keras functional api. 

By the way, I check hyperopt fmin does not support keras functional apis

**Comment by zahs123** [comment]
also havng this issue

**Comment by xliang10** [comment]
Maybe encountered the same problem, and I solved it by:

**Comment by xliang10** [comment]
I may have encountered the same problem but it has been solved.
In my case, I used sklearn.model_selection.cross_val_score to get the score in objective function, but it returned nan because I forgot parsing params `n_estimators` and `max_depth` into int after using hp.quniform. 
An effective solution is to raise errors / verbose=True. I found the problem in my case by using cross_val_score(......, error_score='raise')
